src/plugin.js

```diff
@@ -535,14 +553,17 @@ class VR extends Plugin {
 
     if (this.controls3d) {
       this.controls3d.dispose();
+      this.controls3d = null;
```

Comment:
mostly we needed this change, since we check to see if this variable exists before adding orbit orientation controls, and even with dispose it will still exist.